app/src/main/java/quizapp/volkova/weatherapp/CitiesListActivity.kt

```diff
@@ -0,0 +1,13 @@
+package quizapp.volkova.weatherapp
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+
+class CitiesListActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_cities_list)
+
```

Comment:
The RecyclerView can be initialized here. 

Also, we can pass the result of the selected item back to the MainActivity (using the setResult function).

https://developer.android.com/training/basics/intents/result
https://developer.android.com/reference/android/app/Activity